driver/others/dynamic_arm64.c

```diff
@@ -274,10 +279,12 @@ static gotoblas_t *get_coretype(void) {
           return &gotoblas_CORTEXA73;
         case 0xd0c: // Neoverse N1
           return &gotoblas_NEOVERSEN1;
+#ifndef NO_SVE		      
```

Comment:
Since we're aliasing these above, this isn't necessary to be guarded by `NO_SVE`?